src/test/spec/versioned_api.rs

```diff
@@ -1,100 +1,12 @@
-use std::path::PathBuf;
+use tokio::sync::RwLockWriteGuard;
 
-use tokio::sync::{RwLockReadGuard, RwLockWriteGuard};
-
-use crate::{
-    bson::doc,
-    options::{ServerApi, ServerApiVersion},
-    test::{
-        log_uncaptured,
-        run_single_test,
-        run_spec_test_with_path,
-        spec::unified_runner::TestFile,
-        EventClient,
-        CLIENT_OPTIONS,
-        LOCK,
-    },
-};
+use crate::test::{run_spec_test, LOCK};
 
 use super::run_unified_format_test;
 
 #[cfg_attr(feature = "tokio-runtime", tokio::test)]
 #[cfg_attr(feature = "async-std-runtime", async_std::test)]
 async fn run() {
     let _guard: RwLockWriteGuard<_> = LOCK.run_exclusively().await;
-    run_spec_test_with_path(&["versioned-api"], run_non_transaction_handling_test).await;
-}
-
-#[cfg_attr(feature = "tokio-runtime", tokio::test)]
-#[cfg_attr(feature = "async-std-runtime", async_std::test)]
-async fn run_transaction_handling_spec_test() {
-    let _guard: RwLockWriteGuard<_> = LOCK.run_exclusively().await;
-    let path: PathBuf = [
-        env!("CARGO_MANIFEST_DIR"),
-        "src",
-        "test",
-        "spec",
-        "json",
-        "versioned-api",
-        "transaction-handling.json",
-    ]
-    .iter()
-    .collect();
-    run_single_test(path, &run_unified_format_test).await;
-}
-
-async fn run_non_transaction_handling_test(path: PathBuf, file: TestFile) {
-    if path.ends_with("transaction-handling.json") {
-        log_uncaptured("skipping run_non_transaction_handling_test");
-        return;
-    }
-    run_unified_format_test(file).await
-}
-
-// TODO RUST-817 Remove this test in favor of transaction-handling.json versioned API spec test when
-// transactions are implemented in the unified runner
```

Comment:
This test was a replication of the test in `transaction-handling.json` that I wrote before we had transaction support implemented in the unified test runner. The ticket linked here implies that we would need all of the transactions spec tests ported over to the unified test format to remove this, but we really just needed transactions support in our own runner (which we now have).